docs/compass/08-01-tutorial.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Manage your Applications using Kyma and Compass Console UI
+type: Tutorial
+---
+
+This tutorial present end-to-end use case scenario that shows how to connect an external Application to Compass so that its lambda calls and gets the order services.
```

Comment:
```suggestion
This tutorial presents an end-to-end use case scenario that shows how to connect an external Application to Compass so that its lambda calls and gets the order services. When going through the steps, will switch between two UIs:
- Compass UI where you create a matching between an application, runtime, and scenario
- Console UI where you manage resources used in your application, such as Services, Events, Lambdas, and RoleBindings

```